### PR TITLE
[MM-58442] Migrate tooltips of 'components/drafts/panel/panel_header' to WithTooltip

### PR DESCRIPTION
**Pull request by ahmadJT**
#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/27176
https://mattermost.atlassian.net/browse/MM-58442

#### Release Note

```release-note

```

### CODE REVIEW

File: webapp/channels/src/components/drafts/panel/panel_header.tsx [hunk 1]
```diff
@@ -67,16 +54,18 @@ function PanelHeader({
...
-                            trigger={Constants.OVERLAY_DEFAULT_TRIGGER}
-                            delayShow={Constants.OVERLAY_TIME_DELAY}
+                        <WithTooltip
+                            id='drafts-sync-tooltip'
                             placement='top'
-                            overlay={syncTooltip}
+                            title={{
+                                defaultMessage: 'Updated from another device',
+                                id: 'drafts.info.sync',
+                            }}
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/drafts/panel/panel_header.tsx:63]: We should be using
```
<FormattedMessage
                id='drafts.info.sync'
                defaultMessage='Updated from another device'
            />
            ```

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

**Comment by M-ZubairAhmed** [comment]
@ahmadJT can we get this to finish line

**Review by M-ZubairAhmed — APPROVED** [review]

**Review by yasserfaraazkhan — APPROVED** [review]

**Pull request merged**